srk/src/iteration.ml

```diff
@@ -700,4 +700,132 @@ module SumWedge (A : PreDomainWedge) (B : PreDomainWedge) () = struct
       Right (B.abstract_wedge srk tr_symbols wedge)
 end
 
+(* Computes vector space of invariants over program vars 
```

Comment:
Top-level comments belong in the .mli file.  I can't make sense of the comment though.  I would describe what's going on as a dimension reduction: Given a formula describing a transition system, we compute a formula that describes an isomorphic transition system in a lower-dimensional ambient space.  The example that you give is helpful, though (but x' and y' should also appear free in G).